src/10-Dashboard/DashboardDisplay.tsx

```diff
@@ -30,13 +30,19 @@ const DashboardDisplay = ({navigation}:StackNavigationProps):JSX.Element => {
     const dispatch = useAppDispatch();
     const jwt:string = useAppSelector((state:RootState) => state.account.jwt);
     const partnerPendingUserList:PartnerListItem[] = useAppSelector((state:RootState) => state.account.userProfile.partnerPendingUserList) || [];
-    const [newPartner, setNewPartner] = useState<PartnerListItem|undefined>(undefined);
     const circleInviteList:CircleListItem[] = useAppSelector((state:RootState) => state.account.userProfile.circleInviteList) || [];
     const circleAnnouncementList:CircleAnnouncementListItem[] = useAppSelector((state:RootState) => state.account.userProfile.circleAnnouncementList) || [];
     const newPrayerRequestList:PrayerRequestListItem[] = useAppSelector((state:RootState) => state.account.userProfile.newPrayerRequestList) || [];
     const recommendedContentList:ContentListItem[] = useAppSelector((state:RootState) => state.account.userProfile.recommendedContentList) || [];
 
-  
+    const [newPartner, setNewPartner] = useState<PartnerListItem|undefined>(undefined);
+    const [newPrayerRequests, setNewPrayerRequests] = useState(newPrayerRequestList);
+
+    // if a user deletes a prayer request in the newPrayerRequest list, remove it and re-render
+    useEffect(() => {
+        setNewPrayerRequests(newPrayerRequestList);
+    }, [newPrayerRequestList])
```

Comment:
These `newPrayerRequestList` are recipient Prayer Requests.  Not owned by the logged in user; therefore cannot be deleted.  We did talk about dismissing items on the dashboard; which I think would be more a suppressed backlist design that saves to file.

1. Can you explain the difference between redux `newPrayerRequestList` and local  `newPrayerRequests`?

With redux, I believe defaulting in useState is sufficient.  When redux changes the whole component should re-render as it is subscribed and listening to the redux fields.  If I'm wrong and React Native behaves differently we can keep it.  I see the display lists also update via useEffect.

Reply:
You're right. The change in redux triggers a re-render when newPrayerRequestList changes. The state variables aren't necessary. Reverting back to original code.